app/src/features/authentication/login/services/apiLogin.ts

```diff
@@ -13,8 +13,9 @@ export async function login(user: User) {
   });
 
   const data = await res.json();
-
+  console.log(data);
```

Comment:
remove unused printing

Reply:
ana eh elly mkhly apiLogin edited 3ndi asln XD.

ok, I'll remove it.